pymobiledevice3/services/webinspector.py

```diff
@@ -179,15 +189,23 @@ def get_open_pages(self):
                 apps[self.connected_application[app].name] = self.application_pages[app].values()
         return apps
 
-    def open_app(self, bundle) -> Application:
+    def reload_pages(self):
```

Comment:
Too specific to be a function and doesn't reload pages - just copy the 2 lines to the cli

Reply:
move the function to the webinspector cli file.